scripts/pi-hole/php/auth.php

```diff
@@ -100,19 +100,8 @@ function check_cors()
 
 function check_csrf($token)
 {
-    // Check CSRF token
-    $session_started = function_exists('session_status') ?
```

Comment:
@DL6ER - was this check removed on purpose? See https://github.com/pi-hole/AdminLTE/issues/2350